File: lib/screens/dashboard/widgets/ItemDesign.java
class ItemDesign extends StatelessWidget {
  const ItemDesign( {
        super.key,

//                required this.imagePath,
//                required this.dishName,
//                required this.dishPrice,
    });

    final String imagePath;
    final String dishName;
    final String dishPrice;

    @override
    Widget build(BuildContext context) {
        return Container(
                decoration:BoxDecoration(
                border:Border.all(color:Color(0xFF173033)),
        borderRadius:
        BorderRadius.circular(10)),
        child: const Column(
                crossAxisAlignment:CrossAxisAlignment.center,
                mainAxisAlignment:MainAxisAlignment.center,
                children: [
        Image(
                image:AssetImage(

                imagePath),
                width:69,
                height:69,
          ),
        Text(
                dishName,
                style:TextStyle(color:Color(0xFF89BA2D), fontSize:16),
          ),
        Text(
                dishPrice,
                style:TextStyle(
                color:Color(0xFF89BA2D),
                fontSize:16,
                fontWeight:FontWeight.w700),
          ),
        ],
      ),
    );
    }
}

File: lib/screens/dashboard/widgets/ItemDesign.dart
import 'package:flutter/material.dart';

class ItemDesign extends StatelessWidget {
  final String title;
  final String subtitle;
  final AssetImage imagePath;

  const ItemDesign({
    Key? key,
    required this.title,
    required this.subtitle,
    required this.imagePath,
  }) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Card(
      color: Color(0xFF173033),
      child: Column(
        mainAxisAlignment: MainAxisAlignment.center,
        children: [
          Image(image: imagePath ,width: 80, height: 80,),
          SizedBox(height: 5,),
          Text(title, style: TextStyle(fontSize: 16, fontWeight: FontWeight.bold ,color: Color(0XFF89BA2D)) ),
          Text(subtitle, style: TextStyle(fontSize: 14, color: Color(0XFF89BA2D))),
        ],
      ),
    );
  }
}
